gammapy/maps/geom.py

```diff
@@ -1349,6 +1350,10 @@ def _fill_header_from_axes(self, header):
             else:
                 raise ValueError("Invalid node type {!r}".format(ax.node_type))
 
+            key_interp = "INTERP%i" % idx
```

Comment:
If it's just used once you could avoid the extra local variable since this is still short and readable:
```
header["INTERP%i" % idx] = ax._interp
```

Reply:
Now it is used twice, so keeping a variable name